docs/docs/migrate/keptn/strategy.md

```diff
@@ -80,7 +80,7 @@ Some key points:
 * Keptn provides an operator
   that can observe and orchestrate application-aware workload life cycles.
   This operator leverages Kubernetes webhooks
-  and the Kubernetes scheduler
+  and the default Kubernetes scheduler
```

Comment:
here I would mention scheduling gates instead of the scheduler, as the scheduling gates is the concept we are actively using for this